Tutorial/GettingStartedGuide.md

```diff
@@ -0,0 +1,148 @@
+# Getting Started Guide  
+  
+This guide explains how to package up your trained model as a web service with a predict API that can be scanned using Certifai
+  
+  
+## Table of Contents  
+1. [pre-requisites](#pre-requisites)   
+2. [predict](#predict)  
+3. [api](#api)  
+4. [build-container](#build-container)  
+5. [run-container](#run-container)  
+5. [test-container](#test-container)  
+  
+## pre-requisites  
+  
+1. trained model
+2. [source-to-image (s2i)](https://github.com/openshift/source-to-image)  
+3. [Httpie](https://httpie.org/) (Required to run `make test`)   
+  
+## predict     
+ Create a predict function that takes a message object as function parameter containing data instances to predict against   
+and return list of predictions   
+
+   > refer to function `predict_titanic_survivor` in `titanic_survivor_prediction/predict.py`
+  
+##  api
+  
+Exposing model predict as an api using [s2i](https://github.com/openshift/source-to-image) requires  
```

Comment:
put a link to the environment file so user can see it

Reply:
> Somewhere we need to tell the user that they need to `cd Tutorial/titanic_survivor_prediction` and `pip install -r requirements.txt`. We may also want to take them through setting up a conda env with python 3.6. Currently, I don't think a user who was totally new to this could figure out the steps to let them run the code in the tutorial.
> 
> We may want to have a separate section from the `prerequisites` where we say that we provide a Makefile for convenience (and say what the three commands are). If so then remove `httpie` from prerequisites and add along with `make` in the Makefile section. Also remove the `httpie` example from the test section.

@cdraper-CS so al this happens inside the container using s2i, user does not need to create a local environment or install dependencies.